src/main/java/appeng/container/implementations/ContainerInterfaceTerminal.java

```diff
@@ -57,7 +53,7 @@ public final class ContainerInterfaceTerminal extends AEBaseContainer {
      */
     private static long autoBase = Long.MIN_VALUE;
 
-    private final Multimap<IInterfaceHost, InvTracker> diList = HashMultimap.create();
+    private final Multimap<IInterfaceTerminalSupport, InvTracker> diList = HashMultimap.create();
```

Comment:
nit: Needs a different name, was duality interface list but now we've expanded to a more general term (probably crafting providers).

Reply:
fixed